test-apps/testpurchasesuiandroidcompatibility/build.gradle.kts

```diff
@@ -48,10 +49,29 @@ android {
     }
 }
 
+emerge {
+    // TODO: RevenueCat to set from CircleCi variables
+    apiToken.set(System.getenv("EMERGE_API_TOKEN"))
+
+    vcs {
+        // TODO: RevenueCat to set from CircleCi variables
+        sha.set("")
+        // TODO: RevenueCat to set from CircleCi variables
+        // Should skip setting for main branch uploads
+        baseSha.set("")
```

Comment:
Just to check, this will be the sha for the latest commit in `main` right? I don't think CircleCI gives us this https://circleci.com/docs/variables/? Though we can calculate it through a script of course

Reply:
That is correct, needs to be the commit in the default branch (i.e. main) that the PR is based on. Seems we might be able to get it from GitHub data in CircleCI.

Proposition here: let's add this in a future PR. Most snapshots integrations are best as two steps anyway:
1. Add uploads from all main branch commits. This way we can ensure everything's running properly and we get snapshots for all base builds.
2. In follow up PR, we add support for PR commits. That way diffs we can test that diffs are working properly.

So this PR will be step 1 above, and you or I can follow with step 2 where we add the baseSha just for PR commits. Sound good?

Reply:
Makes a lot of sense thank you!